src/components/AssetSearch/helpers/enrichAsset/enrichAsset.ts

```diff
@@ -0,0 +1,28 @@
+import { Asset } from '@shapeshiftoss/types'
+import { MarketDataState } from 'state/slices/marketDataSlice/marketDataSlice'
+import { AccountRowData } from 'state/slices/portfolioSlice/selectors'
+
+/**
+ * This function enrich the Asset by adding the user cryptoAmount and marketCap for each asset for facilitate sorting.
+ * @param assets
+ * @param rowData
+ * @param marketData
+ * @returns
+ */
+export const enrichAsset = (
+  assets: Asset[],
+  rowData: AccountRowData[],
+  marketData: MarketDataState,
+): Asset[] => {
+  return assets.map(asset => {
+    const fAmount = rowData.find(d => d.assetId === asset.assetId)?.fiatAmount
```

Comment:
Don't use shorthands like `fAmount` instead of `fiatAmount` or `d` instead of `rowData`